packages/bundle-size/package.json

```
@@ -18,9 +19,11 @@
     "find-up": "^5.0.0",
     "glob": "^7.1.2",
     "gzip-size": "^6.0.0",
+    "prettier": "2.2.1",
     "pretty-bytes": "^5.6.0",
     "terser": "^5.5.1",
     "webpack": "^5.21.2",
+    "workspace-tools": "0.16.2",
```

```suggestion
    "workspace-tools": "^0.16.2",
```

These dependencies are public and should use carets.